.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .cache
 dist
+.idea
+.idea/
```

Comment:
Does `.idea` without the backslash not serve the purpose of ignoring the folder as well as it's contents?

Reply:
Might be, you can correct that :)